// thread 'main' panicked at src\middleend\type_checker.rs:1592:17:
// internal error: entered unreachable code
// note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
// 

class Onsug {
    jobozo: bool;
    nzt: Hxpvpvl;
    func ovuc() -> Fvlkzw {
    while (89) {
        let wchssb: Rrsqlvvac = d;
        14;
        true;
        rgsgvlvdi();
}
    while (true) {
        -87;
        return 76;
        break;
        -true > true;
        gaa;
}
    --jlnxha;
}

    func zrwklj(ocyiskdkcx: Sibw, aloczp: Lbqzec) -> Cysypb {
    break;
    if (true) {
        return -60;
        jov;
        break;
        break;
        break;
} else {
        let avgcpxnerw: i64 = abkddmy(6, false, false);
        29.avgcpxnerw;
        continue;
}
    let oiyol: Weuu = bka;
    87.hdvwgsrq;
    return 46;
}

    func opbohnwhc(rmk: Dghpow) -> Kcwyzqjm {
    break;
    let uhx: Crhamvxfn = 54.vigoydtf;
    uhx.qra ^ false;
    continue;
}

    func qnqavkog(moc: Kyieyyvvsm, vhsibdu: i32, dwbrxw: Zo) -> bool {
    let aldxglzh: i32 = -62;
    69.a;
}

    constructor(curlpc: Polpudp, xiywr: Xqvh, nqqhk: i64, qx: Id) {
}

}

func main() -> u32 { return 0; }
